src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingLine.shared.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.ObjectModel;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The Line object is used to describe the lines that are drawn on a <see cref="MauiDrawingView"/>.
+/// </summary>
+public partial class MauiDrawingLine
```

Comment:
Shouldn't this be an implementation of `ILine`? Since this has the same properties. If yes this shouldn't be on `Core`

Reply:
No, it's a native platform control

Reply:
So where the ILine is used?

Reply:
In CommunityToolkit.Maui. See DrawingLine

Reply:
So would not be better to transform the `Line` inside `Core` to be a `Handler`? Then we can use the interface

Reply:
OK, I will simplify it